### PR TITLE
py-iminuit: add new versions 2.8.4

### PR DESCRIPTION
**Pull request by vvolkl**
Adding the latest versions of the 2.X and 1.X release series. Both build fine for me on arch.

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-iminuit/package.py [hunk 1]
```diff
@@ -11,6 +11,8 @@ class PyIminuit(PythonPackage):
 
     pypi = "iminuit/iminuit-1.2.tar.gz"
 
+    version('2.8.4', sha256='4b09189f3094896cfc68596adc95b7f1d92772e1de1424e5dc4dd81def56e8b0')    
+    version('1.5.2', sha256='0b54f4d4fc3175471398b573d24616ddb8eb7d63808aa370cfc71fc1d636a1fd')
     version('1.3.7', sha256='9173e52cc4a0c0bda13ebfb862f9b074dc5de345b23cb15c1150863aafd8a26c')
     version('1.3.6', sha256='d79a197f305d4708a0e3e52b0a6748c1a6997360d2fbdfd09c022995a6963b5e')
     version('1.2', sha256='7651105fc3f186cfb5742f075ffebcc5088bf7797d8ed124c00977eebe0d1c64')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-iminuit/package.py:18]: Can you add that Python 3.6+ is required, at least in the latest version? You can check the `setup.cfg` to see when it was added.

**Review by adamjstewart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-iminuit/package.py (hunk 1 above)
Comment (vvolkl) [reply var/spack/repos/builtin/packages/py-iminuit/package.py:18]: Sure!

**Review by adamjstewart — DISMISSED** [review]

**Comment by adamjstewart** [comment]
Flake8:
```
  var/spack/repos/builtin/packages/py-iminuit/package.py:14: [W291] trailing whitespace
```

**Review by adamjstewart — APPROVED** [review]

**Comment by adamjstewart** [comment]
@spackbot run pipeline

**Pull request merged**